docs/development/updating-components.md

```diff
@@ -0,0 +1,90 @@
+# How to update components?
+
+## Introduction
+
+This is a developer document (not for end users) which explains how to upgrade a particular component in lokoctl. This includes steps to pull config from upstream.
```

Comment:
I'd add a README.md to `development` directory to avoid duplicating `This is a developer document (not for end users)` in every doc in this directory :wink:

Just `This document explains how to upgrade a particular component in lokoctl. This includes steps to pull config from upstream.`